amplify/src/main/java/com/github/stkent/amplify/prompt/DefaultLayoutPromptView.java

```diff
@@ -114,7 +114,7 @@ private void init(@Nullable final AttributeSet attributeSet) {
         typedArray.recycle();
     }
 
-    private static class SavedState extends BaseSavedState {
+    private static class SavedState extends BasePromptView.SavedState {
```

Comment:
This inheritance seems odd when combined with the current implementation of `onSaveInstanceState`:

```java
@Override
protected Parcelable onSaveInstanceState() {
    final Parcelable superState = super.onSaveInstanceState();
    final SavedState savedState = new SavedState(superState);
    savedState.config = config;
    return savedState;
}
```

Feels like we're putting super state in two different places?